common/utilities/string/string-utilities.h

```diff
@@ -8,6 +8,10 @@
 #include <sstream>
 #include <cmath> // std::isfinite
 
+#ifdef WIN32
+#include <Windows.h>
```

Comment:
I think windows-specific functions should go in a windows-specific header...
We shouldn't make every user of string-utilities now also include Windows.h